testing/mod.ts

```diff
@@ -2,6 +2,8 @@
 
 // Do not add imports in this file in order to be compatible with Node.
 
+import { green, red } from "../colors/mod.ts"; // see https://github.com/denoland/deno_std/issues/222
```

Comment:
The comment is unnecessary, please remove.

The comment above 
```
// Do not add imports in this file in order to be compatible with Node.
```
is out-of-date.. we no longer need to keep this file compatible with node since https://github.com/denoland/deno/pull/1749
Please remove this comment too while you're at it.

Reply:
Done

Reply:
Also remove // see https://github.com/denoland/deno_std/issues/222

Reply:
👍